Makefile

```diff
@@ -26,6 +26,7 @@ else
 endif
 K_BIN := $(K_RELEASE)/bin
 K_LIB := $(K_RELEASE)/lib/kframework
+export K_OPTS ?= -Xmx16G -Xss512m
```

Comment:
There is already this line in in the kwasm script:

```
export K_OPTS="${K_OPTS:--Xmx16G -Xss512m}"
```

We could set something similar in the Python script if necessary, rather than put it in the Makefile.